cluster/store_apply.go

```diff
@@ -93,20 +93,19 @@ func (st *Store) Apply(l *raft.Log) interface{} {
 	// schemaOnly is necessary so that on restart when we are re-applying RAFT log entries to our in-memory schema we
 	// don't update the database. This can lead to data loss for example if we drop then re-add a class.
 	// If we don't have any last applied index on start, schema only is always false.
-	schemaOnly := st.lastAppliedIndexOnStart.Load() != 0 && l.Index <= st.lastAppliedIndexOnStart.Load() || st.cfg.MetadataOnlyVoters
+	schemaOnly := l.Index != 0 && l.Index <= st.lastAppliedIndexToDB.Load() || st.cfg.MetadataOnlyVoters
```

Comment:
Is it by design here we have changed `st.lastAppliedIndexOnStart.Load()` with `l.Index` ? Do we instead want to check `lastAppliedIndexToDB` ? 

If last applied index to db is 0, we will never apply any change to it I believe with that change. Or am I wrong here ?

Reply:
yeah made it in purpose to remove, actually that check to to force schemaOnly to true in 1st index


originally 
```
st.lastAppliedIndexOnStart.Load() != 0 && l.Index <= st.lastAppliedIndexOnStart.Load() || st.cfg.MetadataOnlyVoters
```
if `st.lastAppliedIndexOnStart.Load() == 0`,  `l.Index` would never be less than `0`